applications/serial_lte_modem/src/nativetls/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+zephyr_include_directories(.)
+target_sources_ifdef(CONFIG_SLM_NATIVE_TLS app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/slm_at_cmng.c)
```

Comment:
No need for the `ifdef` part I think as this directory is already only included if `CONFIG_SLM_NATIVE_TLS`?